airlock/business_logic.py

```diff
@@ -1785,15 +1810,17 @@ def update_file_in_request(
         ), "File hash does not match manifest.json"
 
         request_file = release_request.get_request_file_from_output_path(relpath)
+        old_group = request_file.group
+        old_filetype = request_file.filetype
 
         for reviewer_username in request_file.reviews:
```

Comment:
When a file is withdrawn, does it keep all its existing reviews? Again, maybe this just needs some more explanation about why we have to do it

Reply:
The behaviour prior to this PR (which is from https://github.com/opensafely-core/airlock/pull/563 ) is that when a file is withdrawn it keeps all its reviews, then if it is subsequently re-added, it loses those reviews. Maybe we should be keeping those reviews?

Reply:
I think current behaviour is OK, but maybe a comment to say so?